tools/check_release.py

```diff
@@ -362,11 +394,22 @@ def get_latest_library_versions(repo_path):
 
     return mbed, mbed_dev
 
+def log_results(lst, title):
+    logging.info(title)   
```

Comment:
Yes. I was thinking something like that. This would make it trivial to grep for failures/skipps/passes in a log file.